[0.224s] Invoking command in '/home/devpc/Desktop/quad_ws/build/quad_interfaces': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/devpc/Desktop/quad_ws/install/quad:/opt/ros/foxy /usr/bin/cmake --build /home/devpc/Desktop/quad_ws/build/quad_interfaces -- -j8 -l8
[0.248s] -- Found ament_cmake: 0.9.8 (/opt/ros/foxy/share/ament_cmake/cmake)
[0.267s] -- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.10", minimum required is "3") 
[0.267s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.437s] -- Found ament_lint_auto: 0.9.6 (/opt/ros/foxy/share/ament_lint_auto/cmake)
[0.493s] -- Found rosidl_default_generators: 1.0.1 (/opt/ros/foxy/share/rosidl_default_generators/cmake)
[0.495s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.499s] -- Found rosidl_adapter: 1.2.1 (/opt/ros/foxy/share/rosidl_adapter/cmake)
[0.624s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.669s] -- Found PythonInterp: /usr/bin/python3 (found version "3.8.10") 
[0.841s] CMake Error at /opt/ros/foxy/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
[0.841s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.841s]   quad_interfaces --arguments-file
[0.841s]   /home/devpc/Desktop/quad_ws/build/quad_interfaces/rosidl_adapter__arguments__quad_interfaces.json
[0.841s]   --output-dir
[0.841s]   /home/devpc/Desktop/quad_ws/build/quad_interfaces/rosidl_adapter/quad_interfaces
[0.841s]   --output-file
[0.842s]   /home/devpc/Desktop/quad_ws/build/quad_interfaces/rosidl_adapter/quad_interfaces.idls)
[0.842s]   returned error code 1:
[0.842s] 
[0.842s]   Error processing 'float32[12] maxAngle' of
[0.842s]   'quad_interfaces/MotionServosCalibration': ''maxAngle' is an invalid field
[0.842s]   name.  It should have the pattern '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$''
[0.842s] 
[0.842s]   Traceback (most recent call last):
[0.842s] 
[0.842s]     File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
[0.842s]       return _run_code(code, main_globals, None,
[0.843s]     File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
[0.843s]       exec(code, run_globals)
[0.843s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.843s]       sys.exit(main())
[0.843s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
[0.843s]       abs_idl_file = convert_to_idl(
[0.843s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.843s]       return convert_msg_to_idl(
[0.843s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.843s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.844s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 515, in parse_message_string
[0.844s]       fields.append(Field(
[0.844s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 353, in __init__
[0.844s]       raise NameError(
[0.844s] 
[0.844s]   NameError: 'maxAngle' is an invalid field name.  It should have the pattern
[0.844s]   '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$'
[0.844s] 
[0.845s] Call Stack (most recent call first):
[0.845s]   /opt/ros/foxy/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.845s]   CMakeLists.txt:37 (rosidl_generate_interfaces)
[0.845s] 
[0.845s] 
[0.845s] -- Configuring incomplete, errors occurred!
[0.845s] See also "/home/devpc/Desktop/quad_ws/build/quad_interfaces/CMakeFiles/CMakeOutput.log".
[0.848s] make: *** [Makefile:748: cmake_check_build_system] Error 1
[0.849s] Invoked command in '/home/devpc/Desktop/quad_ws/build/quad_interfaces' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/devpc/Desktop/quad_ws/install/quad:/opt/ros/foxy /usr/bin/cmake --build /home/devpc/Desktop/quad_ws/build/quad_interfaces -- -j8 -l8
